### ISSUE TITLE
`ollama create` tries to pull model when using quotes in `FROM` line

### ISSUE THREAD

**Issue by savareyhano** [ollama/ollama#4114]
### What is the issue?

Steps to reproduce:

1. Download and install [Ollama v0.1.33 from release page](https://github.com/ollama/ollama/releases/tag/v0.1.33)
2. Download any gguf file and make a modelfile for the gguf file (for example, im using hermes 2 pro):
    Modelfile
    ```
    FROM "./Hermes-2-Pro-Mistral-7B.Q8_0.gguf"
    TEMPLATE """{{ if .System }}<|im_start|>system
    {{ .System }}<|im_end|>
    {{ end }}{{ if .Prompt }}<|im_start|>user
    {{ .Prompt }}<|im_end|>
    {{ end }}<|im_start|>assistant
    """
    PARAMETER stop <|im_start|>
    PARAMETER stop <|im_end|>
    PARAMETER num_ctx 4096
    ```
4. Create the model from the modelfile
    ```bash
    ollama create hermes2-pro-mistral:7b-q8_0 -f ./Modelfile
    ```
5. Run `ollama list` and see if the model is registered from the modelfile (it is not).

Rolled back to [Ollama v0.1.32](https://github.com/ollama/ollama/releases/tag/v0.1.32), and it is now working again so i'm pretty sure this is a problem with the version 0.1.33.

### OS

Windows

### GPU

Nvidia

### CPU

AMD

### Ollama version

0.1.33

**Comment by github-office** [comment]
If you're using windwos, in the Modelfile file, change the file path to C:\\Users\\ABC\\Downloads\\ollama\\qwen1_5-0_5b-chat-q8_0.gguf to try

**Comment by savareyhano** [comment]
all good now, this is fixed with the version [v0.1.34](https://github.com/ollama/ollama/releases/tag/v0.1.34) now it is correctly reading `FROM` using quotes, thanks @jmorganca
